### DISCUSSION TITLE
部署模式

### DISCUSSION THREAD

**Discussion by andoxir** [apache/streampark#3331] [General]
现有一个基于阿里云的flink集群，然后一台独立的机器，想在这台独立的机器上部署streamx（1.2.4版本），然后提交任务到远程的flink集群，这种方案我看按照官网说的是在添加flink cluter里面选择remote模式是吗？以下是我的操作步骤：
1. 在独立机器上下载基础的flink包
2. 在独立的机器上部署streamx，服务已经ok
3. 在添加flink cluster时候，address只要填写远程flink集群的地址就好了吗？
![image](https://github.com/apache/incubator-streampark/assets/16255965/d28584e3-98df-41a9-8058-5cdb70ba6cee)

当然我这里可能网络连接有点问题。

**Comment by wolfboys** [comment]
Just fill in the Web URL address of the Flink cluster. Good luck

**Reply by andoxir** [reply]
I fill the address of http://10.122.42.68:6123, but it does not work.

**Reply by andoxir** [reply]
The network is ok, but it does not work.
